scripts/domaintools.js

```diff
@@ -1,85 +1,38 @@
-function getUrlByParameter(name){
-    var url=window.location.href;
-    var indexOfEnd=url.length;
-    var index=url.indexOf(name);
-    var length=name.length;
-    return url.slice(index+length+1,indexOfEnd);
+function appendToParent (id, item, text_before, parent, show_item, text_after) {
+  if (item) {
+    if (show_item) {
+      parent.append($(id).text(text_before + item + text_after));
+    } else {
+      parent.append($(id).text(text_before + text_after));
+    }
+  }
```

Comment:
If I pass `item=''` , the condition in line 2 of domaintools.js will evaluate to false. The function will simply return.

Reply:
You won't need `if (item) {` at all, you'll remove that as well.

Reply:
`function appendToParent (id, item, text_before, parent, text_after) {
         parent.append($(id).text(text_before + item + text_after));
}`

Is this what you have in mind ?

Reply:
Since we have reached this point and we have only one line, you could drop the function altogether and just call the following in your code.
```
parent.append($(id).text(text_before + item + text_after));
```

Reply:
But if `item` is `null` then some text will still be appended to the parent element
Something like `Email Domain: null`

Reply:
OK, so its necessary to check the `item` with an `if` and we need the function `appendToParent`.

Reply:
Yes we need the `if` in `appendToParent`. Additionally we will need the parameter `show_item`.